ledger/participant-integration-api/src/main/scala/platform/store/dao/events/TransactionsFlatStreamReader.scala

```diff
@@ -161,6 +161,7 @@ class TransactionsFlatStreamReader(
       ids.async
         .addAttributes(
           Attributes.inputBuffer(
```

Comment:
why do we want to specify the buffer size here? Isn't it enough that the following `mapAsync` specifies a buffer size of size `maxParallelPayloadQueries`?

Reply:
Yeah, it might worth trying to remove it completely. Also given that before the async barrier we already has a buffering stage using our `BatchN`. 

Made a note to check what happens if we remove it completely